docs/ldml/tr35.md

```diff
@@ -2323,12 +2323,57 @@ The XML structure is stable over releases. Elements and attributes may be deprec
+There are two kinds of elements in LDML: _rule_ elements and _structure_ elements.
 
-1.  There is no "mixed" content: if an element has textual content, then it cannot contain any elements.
+For structure elements, there are restrictions to allow for effective inheritance and processing:
+
+1.  There is no ["mixed" content](https://www.w3.org/TR/xml/#sec-mixed-content): if an element has textual content, then it cannot contain any elements.
 2.  The [[XPath](#XPath)] leading to the content is unique; no two different pieces of textual content have the same [[XPath](#XPath)].
+3.  An element that has [value attributes](#Definitions) MUST NOT also have have child elements.
+
+The following fictional (non-LDML) example illustrates the first and third restrictions on structure elements:
+
+```xml
+<parent id="abc" someValue="Some Value"> <!-- Disallowed: value attribute AND children -->
+  Here is some text   <!-- Disallowed: Mixed text and children -->
+  <child id="def"/>
+  <child id="ghi"/>
+</parent>
+```
+
+The above format repeats the "Some Value" in the xpath of each child element, making matching and inheritance processing more complex. Instead, this might be structured as:
+
+```xml
+<parent id="abc"> <!-- Allowed: 'id' is distinguishing -->
+  <someValue value="Some Value"/>
+  <text>Here is some text</text>
+  <child id="def"/>
+  <child id="ghi"/>
+</parent>
+```
+
+The following fictional (non-LDML) example illustrates the first and third restrictions on structure elements:
+
+```xml
+<parent id="abc" someValue="Some Value"> <!-- Disallowed: value attribute AND children -->
+  Here is some text   <!-- Disallowed: Mixed text and children -->
+  <child id="def"/>
+  <child id="ghi"/>
+</parent>
+```
+
+The above format repeats the "Some Value" in the xpath of each child element, making matching and inheritance processing more complex. Instead, this might be structured as:
+
+```xml
+<parent id="abc"> <!-- Allowed: 'id' is distinguishing -->
+  <someValue value="Some Value"/>
+  <text>Here is some text</text>
+  <child id="def"/>
+  <child id="ghi"/>
+</parent>
+```
 
-Rule elements do not have this restriction, but also do not inherit, except as an entire block. The rule elements are listed in serialElements in the supplemental metadata. See also _[Section 4.2 Inheritance and Validity](#Inheritance_and_Validity)_. For more technical details, see [Updating-DTDs](https://cldr.unicode.org/development/updating-dtds).
+Rule elements do not have these restrictions, but also do not inherit, except as an entire block. The rule elements are listed in serialElements in the supplemental metadata. See also _[Section 4.2 Inheritance and Validity](#Inheritance_and_Validity)_. For more technical details, see [Updating-DTDs](https://cldr.unicode.org/development/updating-dtds).
```

Comment:
This is old, and we might as well fix it while you are here.

Instead of serialElements we have the DTD annotation @ORDERED
`<!ELEMENT rbnfrule ( #PCDATA ) >`
`   <!--@ORDERED-->`